.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: required
+language: android
+jdk: oraclejdk8
+android:
+ components:
+    - platform-tools
+    - tools
+    - build-tools-26.0.2
+    - android-26
+    - extra-android-m2repository
+before_script:
+ - chmod +x gradlew
+script:
+- ./gradlew assembleDebug --stacktrace
```

Comment:
you should replace this with `- ./gradlew build`
This will build both `debug` and `release` flavors of the app. Also, it will run tests if there are any.

`gradlew assembleDebug` will only build the debug flavor.